src/components/utils/src/threads/async_runner.cc

```diff
@@ -88,6 +92,22 @@ void AsyncRunner::AsyncRunnerDelegate::waitForDelegate() {
   }
 }
 
+void AsyncRunner::AsyncRunnerDelegate::clearDelegateQueue() {
+  if (!delegates_queue_.empty()) {
+    std::queue<threads::ThreadDelegate*> queue_to_delete;
+    delegates_queue_lock_.Acquire();
+    std::swap(delegates_queue_, queue_to_delete);
```

Comment:
@SNiukalov No appropriate header files included

Reply:
@AByzhynar 
Fixed in:https://github.com/LuxoftSDL/sdl_core/pull/41/commits/dff6e3e9b9309028a4371c081b7dc6216bf21bcc